src/sentry/static/sentry/app/views/organizationGroupDetails/reprocessingDialogForm.tsx

```diff
@@ -82,13 +83,31 @@ function ReprocessingDialogForm({group, organization, Header, Body, closeModal}:
           <NumberField
             name="maxEvents"
             label={t('Enter the number of events to be reprocessed')}
-            help={tct(
-              'You can limit the number of events reprocessed in this Issue. If you set a limit, we will reprocess your most recent events, [strong:and the rest will be deleted.]',
-              {strong: <strong />}
+            help={t(
+              'You can limit the number of events reprocessed in this Issue. If you set a limit, we will reprocess your most recent events.'
             )}
             placeholder={t('Reprocess all events')}
             min={1}
           />
+
+          {
+            // XXX(markus): The following field is required but cannot be
+            // effectively populated with a default value. Unless the user
+            // explicitly selects a value, nothing is sent to the server for
+            // whatever reason. Worked around on the server side.
+          }
+
+          <SelectField
+            name="remainingEventsAction"
+            choices={[
+              ['keep', t('Keep remaining events')],
```

Comment:
I would move this below the impact const and reuse it in the ApiForm's initialData prop. Example:

```
const remainingEventsOptions = [
  ['keep', t('Keep remaining events')],
  ['delete', t('Delete remaining events')],
];
```

```
onCancel={closeModal}
          initialData={{
            remainingEventsAction: remainingEventsOptions[0][0],
          }}
```